apps/address-book/app/components/App/Entities.js

```diff
@@ -48,9 +47,12 @@ const Entities = ({ entities, onRemoveEntity }) => {
             entity={entryAddress}
           />,
           <Tag
+            style={{
+              fontWeight: 1000
+            }}
```

Comment:
Does the `css` prop also work with `Tag`? I think in general we favor that over `style`, if the component supports it. That would let us make it one line, too: `css="font-weight: 1000"`

Also, does this accomplish the same thing as `font-weight: bold`? Or is it actually bolder than that?